plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -197,9 +199,6 @@ public static ReaderPageSource createPageSource(
             ParquetReaderOptions options,
             Optional<ParquetWriteValidation> parquetWriteValidation)
     {
-        // Ignore predicates on partial columns for now.
```

Comment:
I will try to implement a unit test for this line.

Reply:
I will try use localTrinoQueryRunner and exam actual row blocks loaded. working on it.

Reply:
added test using baseHiveConnectorTest